### ISSUE TITLE
How do I prepare zh-en's vocab, and should the word of the 2 correspond to each other?What are the rules for vocab's file?

### ISSUE THREAD

**Issue by zero1207** [tensorflow/nmt#115]

**Comment by ssokhey** [comment]
Use the vocab generator:

[https://github.com/google/seq2seq/blob/master/bin/tools/generate_vocab.py](url)

Generally, the vocabulary file should contain all the unique words with frequency in it. So, there will be two vocabulary files; `vocab.zh` & `vocab.en`.

**Comment by kingkf** [comment]
the input of the script is my train data?

**Comment by ssokhey** [comment]
@kingkf Yes. Also you should specify the output. Make sure you remove the frequency counts coming along with vocabulary. Let me know if you need any help.

**Comment by kingkf** [comment]
@ssokhey thanks!
